docs/docs/adding-react-components.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Adding React Components
+---
+
+This guide covers how to add React components to your Gatsby site.
+
+## React components
+
+React components are prebuilt elements or groups of elements that can be used to prevent repeating yourself in your code.
+
+Components also offer the ability to be customized using "props" (properties). Props can be of any Javascript type, such as boolean, string, object, array or almost anything you can think of.
+
+For example, you could use a component for Buttons on your site. This is because they would likely be used multiple times across pages and would need different labels each time.
+
+## Importing React components
+
+In Gatsby, when using React components, you import and use them just like you would in normal React:
+
+```jsx
+import React from "react"
+import { Link } from "gatsby"
+
+export default () => (
+  <div>
+    <Link to="/contact/">Contact</Link>
+  </div>
+)
+```
+
+### Importing third-party components
+
+Just like React, Gatsby also supports third-party components and libraries. You can install a third-party component or library via your package manager (either `yarn` or `npm`).
+
+Here's a short example on adding a third-party component to your site.
+
+First, you have to install the component or library's package via a package manager. It's recommended not to mix package managers (don't use npm and yarn at the same time).
+
+```shell
+// Install material-ui with npm
+npm install @material-ui/core
+
+// Or with yarn
```

Comment:
Same as above -- we recommend `npm` and use it consistently in the docs, so the yarn instructions should be removed.